documentation/decisions/adr/000N-ci-cd-task-runner.md

```diff
@@ -0,0 +1,32 @@
+# Choose a Task Runner for the CI / CD Pipeline
+
+- **Status:** Draft <!-- REQUIRED -->
+- **Last Modified:** 2023-06-29 <!-- REQUIRED -->
+- **Related Issue:** [ADR: Task Runner and CI / CD interface #92
+](https://github.com/HHS/grants-api/issues/92) <!-- RECOMMENDED -->
+- **Deciders:** TBD <!-- REQUIRED -->
```

Comment:
If you could fill this out before merging, that would be great!

Reply:
Done!